.infrastructure/ingress/ingress.yml

```
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: todoapp-ingress
+  namespace: todoapp
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
+spec:
+  rules:
+    - http:
+        paths:
+          - pathType: Prefix
+            path: /blue(/|$)(.*)
```

```suggestion
            path: /(|$)(.*)
```
Краще на blue накласти рівень абстракції через вираз.

---

не дуже зрозумів
чому пропадає "/" у виразі?

---

`(|$)` This capturing group means "an empty string or the end of the string".